Xamarin.Forms.Core/VisualElement.cs

```diff
@@ -100,6 +100,34 @@ public partial class VisualElement : Element, IAnimatable, IVisualElementControl
 
 		public static readonly BindableProperty FlowDirectionProperty = BindableProperty.Create(nameof(FlowDirection), typeof(FlowDirection), typeof(VisualElement), FlowDirection.MatchParent, propertyChanged: FlowDirectionChanged);
 
+		public static readonly BindableProperty TabIndexProperty =
+			BindableProperty.Create(nameof(TabIndex),
+									typeof(int),
+									typeof(View),
```

Comment:
Did you mean for this to be `VisualElement`, or did you mean to put these properties on `View`? `View` might be a better choice. I don't think it makes sense for `Page`s to have a `TabIndex`/`TabStop`

Reply:
Oh. It's mistake. I mean here `VisualElement`.
I originally made for the `View`. But in `UWP` all UI elements have a `TabIndex/TabStop` include Pages.